### PR TITLE
#559044: [SXA] fixed rendering dynamic placeholders for splitter components

### PR DESCRIPTION
**Pull request by matkovskyi**
## Description / Motivation
Added pattern and checking for rendering Dynamic placeholders. 

There was bug with rendering splitter components with fix Dynamics placeholders. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by StanislavStativa — APPROVED** [review]

File: packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx [hunk 1]
```diff
@@ -12,6 +12,14 @@ import {
 import { convertAttributesToReactProps } from '../utils';
 import { HiddenRendering, HIDDEN_RENDERING_NAME } from './HiddenRendering';
 
+/** These patterns need for right rendering Dynamic placeholders.
```
Comment (illiakovalenko) [inline packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx:15]: Comment should on the next line

**Review by illiakovalenko — CHANGES_REQUESTED** [review]
Please, add unit tests to cover new case

**Comment by matkovskyi** [comment]
> Please, add unit tests to cover new case
I added unit test

**Comment by illiakovalenko** [comment]
@matkovskyi Build is failed, please, fix issues first

**Comment by matkovskyi** [comment]
> @matkovskyi Build is failed, please, fix issues first

fixed! thanks)

**Review by illiakovalenko — APPROVED** [review]

**Pull request merged**